src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -13,6 +13,12 @@ import sbtrelease.ReleasePlugin.autoImport._
 import xerial.sbt.Sonatype
 import xerial.sbt.Sonatype.autoImport._
 
+private object ScalaVersions {
```

Comment:
Maybe this should be public so we can use it in other projects that use interplay.